core/server/services/auth/session/index.js

```diff
@@ -0,0 +1 @@
+module.exports = require('./middleware');
```

Comment:
I am curious, why you decided that this index file exports the middleware file straight?🤔 

There are two files in this package: store and middleware. 

---

For reference: https://github.com/TryGhost/Ghost/blob/2.1.3/core/server/services/routing/index.js
Async loading on runtime and on need. It's currently a common pattern. Brings package notation and consistency.

Reply:
The store is private to the service so doesn't need to be exported. 

As for exporting the middleware straight, the ghost server cannot start without this middleware, so we don't gain anything by deferring the load of it - if anything this would slow down bootstrap because of the extra fn call in the stack and bytes to load!

If that is the way we do all index files though, I can update it :+1: